test/components/Vizr_test.js

```diff
@@ -133,7 +136,7 @@ describe('Vizr.vue', () => {
     describe('when the chart is updated', () => {
       it('persists the new configuration', done => {
         // second chart config will be replaced
-        const updatedChart = Object.assign({}, exampleChartConfig[1]);
+        const updatedChart = Object.assign({}, exampleChartConfig.charts[1]);
```

Comment:
Another weird test bug. Because `exampleChartConfig[1]` is undefined, `updatedChart` is an empty object. This test should have tripped the new chart logic instead of the updated chart logic, and yet it didn't.